### DISCUSSION TITLE
Terraform Deploy Errors

### DISCUSSION THREAD

**Discussion by jvidamins** [twilio-professional-services/flex-project-template#678] [Q&A] [answered]
Got the following errors... Anyone have any advice on how to fix?

![twilio1](https://github.com/user-attachments/assets/0b679a95-5587-4fd8-a3ba-7c16c97a6457)
![twilio2](https://github.com/user-attachments/assets/535d4090-ce0f-4bd5-823f-d8c904ecb87b)

**Comment by jvidamins** [comment, marked as answer]
got if figured out. I had to delete every service that was already auto-installed when I created Flex. Then I checked all 3 boxes when deploying, not just the top 2.

**Reply by dremin** [reply]
Hi @jvidamins! Are you saying that when you created the Flex account, it came with some serverless services pre-installed?

**Comment by MauricioOlitel** [comment]
Hi! I’m running into the same Terraform deploy errors as in this thread.

What I tried:
- Deleted pre-existing/auto-created Serverless services in the Flex account
- Re-ran deploy with all 3 boxes checked


Questions:
1) Which specific auto-created services should be deleted (names/types)?
2) Is there any known ordering/dependency (e.g., remove conversations / taskrouter / serverless first)?
3) Any recommended cleanup steps before re-deploy?

Thanks!

**Reply by dremin** [reply]
Hi @MauricioOlitel! I found an issue and this should be fixed by #820; feel free to try out the fix if you'd like and let me know if it solves the issue for you.
